docs/plugins/repository-s3.asciidoc

```diff
@@ -190,8 +190,12 @@ The following settings are supported:
 
 `storage_class`::
 
-    Sets the S3 storage class type for the backup files. Values may be
+    Sets the S3 storage class for snapshot repository objects. Values may be
```

Comment:
I think I'd prefer "objects stored in the snapshot repository" instead of "snapshot repository objects"?